obspy/imaging/spectrogram.py

```diff
@@ -165,10 +165,7 @@ def spectrogram(data, samp_rate, per_lap=0.9, wlen=None, log=False,
     halfbin_time = (time[1] - time[0]) / 2.0
     halfbin_freq = (freq[1] - freq[0]) / 2.0
 
-    # argument None is not allowed for kwargs on matplotlib python 3.3
```

Comment:
Is Python the problem here or Matplotlib?

Reply:
I assumed matplotlib was the problem.